ansible_runner/runner_config.py

```diff
@@ -220,7 +222,9 @@ def prepare_command(self):
         and if not calls :py:meth:`ansible_runner.runner_config.RunnerConfig.generate_ansible_command`
         """
         try:
-            self.command = self.loader.load_file('args', string_types)
+            cmdline_args = self.loader.load_file('args', string_types)
```

Comment:
Or maybe we should do `self.command = self.loader.load_file('args', list)` and expect the `args` file to be a json array?